modules/virt-about-datavolume-gc.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-deleting-datavolumes.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-datavolume-gc_{context}"]
+
+= About data volume garbage collection
+
+Before {VirtProductName} release 4.12, you used an `oc` command to remove a data volume. This command removes the connection between the data volume and the persistent volume claim (PVC). However, using this command does not always provide the expected result. For example, Kubevirt might attempt to recreate a data volume that you removed if Kubevirt had initially created the volume.
```

Comment:
Any time you use `oc delete` to delete a DV that owns a PVC it results in the PVC being deleted.  Therefore, referencing manual DV deletion here is not really appropriate.  GC is different from deleting a DV because when garbage collecting DVs the system breaks the ownership relation between the DV and PVC first.  When the DV is garbage collected, the PVC is retained and things continue to work as expected.

Reply:
OK, cool. I wasn't sure if we should even be telling people to use `oc delete`. It seems we should not. And as this doc is only being picked to 4.12, we don't have to mention version-specific behavior, which is cleaner and less confusing for the user.

Revised to:
_During automated data volume garbage collection, the link between the data volume and the persistent volume claim (PVC) is broken. After the data volume is deleted by the garbage collector, the PVC is retained. 

The PVC continues to transparently manage data volumes, eliminating the need for you to manually delete data volumes._